crypto/evp_extra/evp_asn1.c

```diff
@@ -70,6 +70,14 @@
 #include "internal.h"
 #include "../dilithium/internal.h"
 
+// The function parse_key_type takes the algorithm cbs sequence |cbs| and
```

Comment:
```Suggestion
// parse_key_type takes the algorithm cbs sequence |cbs| and
```

Reply:
added in [e419b99](https://github.com/aws/aws-lc/pull/1963/commits/e419b9944904e8e30496d5a250b696997bc7e3f0)